MyWorkout.Web/Controllers/ExercisesController.cs

```diff
@@ -90,7 +90,7 @@ await _unitOfWork.ExerciseRepository
 
             await _unitOfWork.Save();
 
-            return RedirectToAction(nameof(Index));
+            return RedirectToAction("WorkoutDay","Details",exercise);
```

Comment:
https://stackoverflow.com/questions/10785245/redirect-to-action-in-another-controller

Here is signature of the method
RedirectToRouteResult RedirectToAction (string actionName, string controllerName, object routeValues);

Your should use
RedirectToAction(nameof(ExcerciseController.Details),"WorkoutDays", exercise.WorkoutDayId);